volumes-web-app/rockcraft.yaml

```diff
@@ -4,117 +4,133 @@ description: |
   This web app is responsible for allowing the user to manipulate PVCs in their Kubeflow
   cluster. To achieve this it provides a user friendly way to handle the lifecycle of PVC
   objects.
-version: v1.7.0_1
+version: "1.8.0"
 license: Apache-2.0
-base: ubuntu:20.04
+base: ubuntu@22.04
+platforms:
+  amd64:
+run-user: _daemon_
 
 services:
-  serve:
-    override: replace
+  kubeflow-volumes:
+    override: merge
     summary: "volumes service"
-    command: gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app
+    command: '/bin/bash -c "gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app"'
+    working-dir: "/src/"
     startup: enabled
-    user: ubuntu
     environment:
       NODE_ENV: production
-platforms:
-  amd64:
 
 parts:
   security-team-requirement:
     plugin: nil
     override-build: |
       mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
-      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
 
   backend:
     plugin: nil
     source: https://github.com/kubeflow/kubeflow
     source-type: git
-    source-tag: v1.7-branch  # upstream branch
+    source-tag: v1.8-branch
     source-depth: 1
     build-packages:
       - python3-venv
       - python3-setuptools
       - python3-pip
     override-build: |
       python3 -m pip install wheel
-      cd components/crud-web-apps/common/backend
+      # Create a directory that holds the src of the backend copied
+      # from the CRAFT_PART_SRC directory to avoid bringing files that
+      # could affect the build of the backend wheel.
+      # This measure helps keeping the rockcraft and the Dockerfile files
+      # as similar as possible in terms of the operations they perform.
+      # This is replicated in other steps for the frontend.
+      mkdir $CRAFT_STAGE/backend-src && cd $CRAFT_STAGE/backend-src
+      cp -r $CRAFT_PART_SRC/components/crud-web-apps/common/backend/* $CRAFT_STAGE/backend-src/
       python3 setup.py bdist_wheel
-      cp dist/kubeflow-1.1-py3-none-any.whl $CRAFT_STAGE
```

Comment:
So we decided to remove this but this means the built wheel stays in build phase for ROCK and wont be available in the final image. I think we need to keep this.